app/features/utils/functions.js

```diff
@@ -54,3 +54,36 @@ export function normalizeServerURL(url: string) {
 export function openExternalLink(link: string) {
     window.jitsiNodeAPI.shellOpenExternal(link);
 }
+
+
+/**
+ * Get URL, extract room name from it and create a Conference object.
+ *
+ * @param {string} inputURL - Combined server url with room separated by /.
+ * @returns {Object}
+ */
+export function createConferenceObjectFromURL(inputURL: string) {
+    let room;
+
+    // Remove trailing slash if exists.
+    if (inputURL.substr(-1) === '/') {
+        inputURL = inputURL.substr(0, inputURL.length - 1); // eslint-disable-line no-param-reassign
+    }
+
+    // If we have more then a room name here,
+    // we assume that the last one is a room name.
+    if (inputURL.includes('/')) {
+        room = inputURL.split('/').pop();
+    } else {
+        room = inputURL;
+    }
+
+    // Don't navigate if no room was specified.
+    if (!room) {
+        return;
+    }
+
+    return {
+        room
```

Comment:
You only return the room here but not the server url. So it will default to the default serverURL.

Reply:
Yep, please port this logic here: https://github.com/jitsi/jitsi-meet-electron/pull/263/files#diff-8265ab18888d94efa59fa6ff246f1688L128

Reply:
Thanks for testing, Annika!
Is there any documentation about how it was expected to be working? I was unable to find any. 
I did see some comments mentioning server option in the app link, but this seemed like a security risk. It makes phishing attacks easier by redirecting users to call server they were not configured to use.
If this is the expected way to use the app link, then there should be some option for admins and users to protect themselves from potential phishing attempts.

Reply:
Fail point. The applink could just take the room into account. Then it would ue the configured server in the app. The problem with your change is that hanndling of the server URL seems to be ccompletely gone. Users should still be able to use the configured server.

Reply:
It caused lots of confusion for my users that the app behaviour changed to not being able to provide a different server than the configured one. So at least in the App input field the server should be treated as before.

Reply:
Thank you both for the feedback. Could you please provide example format/links how it's expected to be working. As the previously linked code sample was commented in code review as "not working", then I wouldn't trust that as an example.
PS! @saghul, I'm bit confused about your comment regards security. Could you please maybe clarify your thinking please, thanks! :)

Reply:
Previously you could enter https:///somejitsiserver/room and it would take that as an input and also connect to that server.

Now it's also taking that input but strips the server name and connects straight to the configured one.

Reply:
@kempu My comment was about the potential security implications of exposing that.

Reply:
Thanks, everybody for the help. Hope the updated code suits better :)